apps/vc-api/src/vc-api/workflows/dtos/create-workflow-request.dto.ts

```diff
@@ -0,0 +1,41 @@
+import { IsString, IsObject, ValidateNested, IsOptional } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { WorkflowStepDto } from './workflow-step.dto';
+
+export class Steps {
+    [key: string]: WorkflowStepDto;
```

Comment:
I think a more meaning full name for the key might be better
```suggestion
    [stepId: string]: WorkflowStepDto;
```